src/database/migrations/1745439840451-create-college-table.ts

```diff
@@ -0,0 +1,36 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class CreateCollegeTable1745439840451 implements MigrationInterface {
+    name = 'CreateCollegeTable1745439840451'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "college" DROP COLUMN "provost"`);
+        await queryRunner.query(`ALTER TABLE "college" DROP COLUMN "year"`);
+        await queryRunner.query(`ALTER TABLE "college" ADD "yearEstablished" date NOT NULL`);
```

Comment:
why yearEstablished when the pattern in the file is snake case. should be year_established